gradlew

```diff
@@ -129,6 +130,7 @@ fi
 if [ "$cygwin" = "true" -o "$msys" = "true" ] ; then
     APP_HOME=`cygpath --path --mixed "$APP_HOME"`
     CLASSPATH=`cygpath --path --mixed "$CLASSPATH"`
+    
```

Comment:
extra new lines in multiple places or intentional?

Reply:
This is just the script that Gradle drops by itself (`./gradle wrapper`). I didn't change anything.